cmd/dhcpserver/list.go

```diff
@@ -49,7 +49,15 @@ var listCmd = &cobra.Command{
 		}
 
 		table := utils.NewTable()
-		table.Render(dhcpservers, nil)
+		table.SetHeader([]string{"ID", "Network ID", "Network Name", "Status"})
+		if len(dhcpservers) != 0 {
+			for _, dhcpserver := range dhcpservers {
+				table.Append([]string{*dhcpserver.ID, *dhcpserver.Network.ID, *dhcpserver.Network.Name, *dhcpserver.Status})
```

Comment:
We observed that when the dhcpServer.Status is building the dhcpserver.network.id will be nil, So it may lead to panic, So better to handle nil check before de referring it.

Reply:
Agreed, added a check to skip populating Network ID and Network Name, as long as they point to `nil`.
```+--------------------------------------+--------------------------------------+----------------------------------------------------+--------+
|                  ID                  |              NETWORK ID              |                    NETWORK NAME                    | STATUS |
+--------------------------------------+--------------------------------------+----------------------------------------------------+--------+
| 0482f18a-288b-41aa-bd9e-4d20f0492c1e |                                      |                                                    | BUILD  |
| 9b790cd2-33d2-4b16-ac5c-f874239d8dfa | 907e1505-7835-4538-bcfc-fa6393909c14 | DHCPSERVERadef8e645d764a6396c812f680e47120_Private | ACTIVE |
| 8809fdc4-1892-42ab-99d1-6b707b5aeda2 | c4eebfdc-b744-426b-a7c2-bfce6c970fcb | DHCPSERVERee20e009ae7b4696b5f9a044ecdea05c_Private | ACTIVE |
```

Additionally addressed the comment from the [issue](https://github.com/ppc64le-cloud/pvsadm/issues/590#issuecomment-2060347790), to display Network Name, IP - MAC and status as the rest of the fields are available from the `list` command. : 
```
+----------------------------------------------------+----------+--------+
|                    NETWORK NAME                    | IP - MAC | STATUS |
+----------------------------------------------------+----------+--------+
| DHCPSERVER78756806c6bf4f41892ec04a88e1a595_Private |          | ACTIVE |
+----------------------------------------------------+----------+--------+
```